shell/server/installer.js

```diff
@@ -385,6 +388,10 @@ AppInstaller = class AppInstaller {
             (manifest.appTitle && manifest.appTitle.defaultText), manifest.appVersion,
             (manifest.appMarketingVersion && manifest.appMarketingVersion.defaultText));
         }
+
+        if (_this.isPreinstalled) {
```

Comment:
I guess this means that any user accepting an update notification for a pre-installed app updates it for all users who have the app pre-installed?